ui/example/type-subtype-value/config/config.json

```diff
@@ -7,5 +7,32 @@
       "startProperty": "startDate",
       "endProperty": "endDate"
     }
+  },
+  "graph": {
+    "style": {
+      "edges": {
+        "BasicEdge": {
+          "line-color": "#00a8ff",
+          "target-arrow-color": "#00a8ff",
+          "text-outline-color": "#00a8ff"
+        }
+      },
+      "vertexTypes": {
+        "vertex": {
+          "background-image": "app/img/material-icons/filter.svg",
+          "text-valign": "top"
+        }
+      },
+      "fieldOverrides": {
```

Comment:
I think fieldOverrides should be a nested object inside vertexTypes/vertexType/
e.g:
```json
"vertexTypes": {
	"vertex": {
		"background-image": "app/img/material-icons/filter.svg",
		"text-valign": "top",
		"fieldOverrides": {
			"type": {
				"t1": {
					"background-image": "app/img/material-icons/filter_1.svg"
				},
				"t2": {
					"background-image": "app/img/material-icons/filter_2.svg"
				}
			}
		}
	}
}
```